arlas-tests/src/test/java/io/arlas/server/rest/explore/AbstractFilteredTest.java

```diff
@@ -946,108 +955,98 @@ public void testInvalidFilterParameters() throws Exception {
         handleInvalidParameters(header(request.filter));
 
-        request.filter.notgwithin = Arrays.asList(new MultiValueFilter<>("foo"));
+        request.filter.f = Arrays.asList(new MultiValueFilter<>(new Expression("geo_params.geometry", OperatorEnum.notwithin, ("foo"))));
         handleInvalidParameters(post(request));
-        handleInvalidParameters(get("notgwithin", request.filter.notgwithin.get(0).get(0)));
+        handleInvalidParameters(get("f", request.filter.f.get(0).get(0).toString()));
         handleInvalidParameters(header(request.filter));
-        request.filter.notgwithin = null;
 
-        request.filter.notgwithin = Arrays.asList(new MultiValueFilter<>("POLYGON((10 10,10 -10,0 -10))"));
+        request.filter.f = Arrays.asList(new MultiValueFilter<>(new Expression("geo_params.geometry", OperatorEnum.notwithin, "POLYGON((10 10,10 -10,0 -10))")));
         handleInvalidParameters(post(request));
-        handleInvalidParameters(get("notgwithin", request.filter.notgwithin.get(0).get(0)));
+        handleInvalidParameters(get("f", request.filter.f.get(0).get(0).toString()));
         handleInvalidParameters(header(request.filter));
-        request.filter.notgwithin = null;
 
         //GINTERSECT
-        request.filter.gintersect = Arrays.asList(new MultiValueFilter<>("POLYGON((10 10,10 -10,0 -10))"));
+        request.filter.f = Arrays.asList(new MultiValueFilter<>(new Expression("geo_params.geometry", OperatorEnum.intersects, "POLYGON((10 10,10 -10,0 -10))")));
         handleInvalidParameters(post(request));
-        handleInvalidParameters(get("gintersect", request.filter.gintersect.get(0).get(0)));
+        handleInvalidParameters(get("f", request.filter.f.get(0).get(0).toString()));
         handleInvalidParameters(header(request.filter));
 
-        request.filter.gintersect = Arrays.asList(new MultiValueFilter<>("foo"));
+        request.filter.f = Arrays.asList(new MultiValueFilter<>(new Expression("geo_params.geometry", OperatorEnum.intersects, "foo")));
         handleInvalidParameters(post(request));
-        handleInvalidParameters(get("gintersect", request.filter.gintersect.get(0).get(0)));
+        handleInvalidParameters(get("f", request.filter.f.get(0).get(0).toString()));
         handleInvalidParameters(header(request.filter));
-        request.filter.gintersect = null;
 
-        request.filter.gintersect = Arrays.asList(new MultiValueFilter<>("POLYGON((1000 10000,10 -10,0 -10,1000 10000))"));
+        request.filter.f = Arrays.asList(new MultiValueFilter<>(new Expression("geo_params.geometry", OperatorEnum.intersects, "POLYGON((1000 10000,10 -10,0 -10,1000 10000))")));
         handleInvalidParameters(post(request));
-        handleInvalidParameters(get("gintersect", request.filter.gintersect.get(0).get(0)));
+        handleInvalidParameters(get("f", request.filter.f.get(0).get(0).toString()));
         handleInvalidParameters(header(request.filter));
-        request.filter.gintersect = null;
 
```

Comment:
It will be good to add a query geopoint:intersects:validGeo to point out that this operation in not supported
```java
request.filter.f = Arrays.asList(new MultiValueFilter<>(new Expression("geo_params.geometry", OperatorEnum.intersects, "POLYGON((-50 50,-20 50, -20 -50, -50 -50,-50 50))")));
handleInvalidParameters(post(request));
handleInvalidParameters(get("f", request.filter.f.get(0).get(0).toString()));
handleInvalidParameters(header(request.filter));
```

Reply:
ok